### DISCUSSION TITLE
Dropdown menu scrolls with whole page

### DISCUSSION THREAD

**Discussion by Ivor808** [JedWatson/react-select#4981] [Q&A]
Hello!

We are using react-select for our drop down menus. I've ran into an issue where if the main page is scrolled while a dropdown is open, the dropdown moves with the scroll. Expected behavior is that the dropdown should keep its positioning, even if the main page is scrolled down or up. Lastly, if you hover over the dropdown, it snaps it back into its 'correct' position. Any idea what could cause an issue like this?

**Comment by saimageshvar** [comment]
Any solution/ updates on this? @Ivor808

**Reply by Ivor808** [reply]
Sadly not.
